docs/developer/solutions/deploy.rst

```diff
@@ -0,0 +1,23 @@
+Deployment And Experimenting
+============================
+
+Given the solution ISO is well generated, A script utiliy has been added to enable solution install and removal
```

Comment:
Not sure about the use of `well` here, maybe `correctly` is better.

`A script` => why `A` instead of `a`? You're not starting a new sentence.
And talking about sentence, you're missing the final period.

Also, long line should be wrapped (apply to the lines below as well).